effects/compressor/compressor.h

```diff
@@ -67,7 +67,7 @@ class Compressor : public Effect
       unsigned int count;
       float as[A_TBL];
 
-      // The blanace between the RMS and peak envelope followers.
+      // The blance between the RMS and peak envelope followers.
```

Comment:
Wrong before and after, needs to be `balance'

Reply:
ooops, fixed in 7d0f10a